site/tests/integration/blockpage.test.ts

```diff
@@ -0,0 +1,113 @@
+import { expect, test } from "playwright-test-coverage";
+
+import blocksData from "../../blocks-data.json";
```

Comment:
This import breaks linting because the file is not checked-in in git: https://github.com/blockprotocol/blockprotocol/runs/7455330240?check_suite_focus=true#step:9:54

You can use `await fs.readJson()` via `fs-promise` instead.

Reply:
Linting breaks because blocksData isn't properly typed when imported. Typed it in [5adad49](https://github.com/blockprotocol/blockprotocol/pull/450/commits/5adad49fdf2865ff470f34040be638718ea3a2aa)